docs/integrations/sources/file.md

```diff
@@ -5,71 +5,99 @@ This page contains the setup guide and reference information for the Files sourc
+
+#### HTTPS: Public Web [Default]
+- `User-Agent` (Optional)
+
+Set this to active if you want to add the [User-Agent header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) to requests (inactive by default).
+
+#### GCS: Google Cloud Storage
+- `Service Account JSON` (Required for **private** buckets) 
+
+To access **private** buckets stored on Google Cloud, this connector requires a service account JSON credentials file with the appropriate permissions. A detailed breakdown of this topic can be found at the [Google Cloud service accounts page](https://cloud.google.com/iam/docs/service-accounts). Please generate the "credentials.json" file and copy its content to this field, ensuring it is in JSON format. **If you are accessing publicly available data**, this field is not required.
+
+#### S3: Amazon Web Services
+- `AWS Access Key ID` (Required for **private** buckets)
+- `AWS Secret Access Key` (Required for **private** buckets)
+
+To access **private** buckets stored on AWS S3, this connector requires valid credentials with the necessary permissions. To access these keys, refer to the 
+[AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+More information on setting permissions in AWS can be found 
+[here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html). **If you are accessing publicly available data**, these fields are not required.
+
+#### AzBlob: Azure Blob Storage
+- `Storage Account` (Required)
+
+This is the globally unique name of the storage account that the desired blob sits within. See the [Azure documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) for more details.
+
+**If you are accessing private storage**, you must also provide _one_ of the following security credentials with the necessary permissions:
+
+- `SAS Token`: [Find more information here](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview).
+- `Shared Key`: [Find more information here](https://learn.microsoft.com/en-us/rest/api/storageservices/authorize-with-shared-key).
+  
+#### SSH: Secure Shell / SCP: Secure Copy Protocol / SFTP: Secure File Transfer Protocol
+- `Host` (Required)
+  
+Enter the _hostname_ or _IP address_ of the remote server where the file trasfer will take place.
+- `User` (Required)
+  
+Enter the _username_ associated with your account on the remote server.
+- `Password` (Optional) 
+  
+**If required by the remote server**, enter the _password_ associated with your user account. Otherwise, leave this field blank.
+- `Port` (Optional) 
+
+Specify the _port number_ to use for the connection. The default port is usually 22. However, if your remote server uses a non-standard port, you can enter the appropriate port number here.
+
+<!-- env:local -->
```

Comment:
I'm not sure `env:local` is a thing -- did you see it somewhere else? the only two i know of are oss and cloud

Reply:
This should probably just be oss since cloud doesn't support local

Reply:
Whoops my bad! I just got mixed up with the env name for oss 🤦‍♂️